codecs/av1_packet.go

```diff
@@ -0,0 +1,116 @@
+package codecs
+
+import (
+	"github.com/pion/rtp/v2/pkg/obu"
+)
+
+const (
+	zMask     = byte(0b10000000)
+	zBitshift = 7
+
+	yMask     = byte(0b01000000)
+	yBitshift = 6
+
+	wMask     = byte(0b00110000)
+	wBitshift = 4
+
+	nMask     = byte(0b00001000)
+	nBitshift = 3
+)
+
+// AV1Payloader payloads AV1 packets
+type AV1Payloader struct{}
+
+// Payload fragments a AV1 packet across one or more byte arrays
+func (p *AV1Payloader) Payload(mtu uint16, payload []byte) [][]byte {
+	return nil
+}
+
+// AV1Packet represents a depacketized AV1 RTP Packet
+//
+//  0 1 2 3 4 5 6 7
+// +-+-+-+-+-+-+-+-+
+// |Z|Y| W |N|-|-|-|
+// +-+-+-+-+-+-+-+-+
+//
+// https://aomediacodec.github.io/av1-rtp-spec/#44-av1-aggregation-header
+type AV1Packet struct {
+	// Z: MUST be set to 1 if the first OBU element is an
+	//    OBU fragment that is a continuation of an OBU fragment
+	//    from the previous packet, and MUST be set to 0 otherwise.
+	Z byte
```

Comment:
I think single bit field is better to be `bool` like
https://github.com/pion/rtp/blob/b08333e6c3a2db0ec7c388c81541d91d78a9ae70/codecs/vp9_packet.go#L115-L153

Reply:
Fixed!